docs: add example to the "Customizing fixture options" section (e.g. for browser_context_args)

Hi, I'm complete new to pytest/playwright and have only small experiences with pytohn. 
That means, my question is propably more a general syntax/understanding problem as a playwright issue. But hopefully somebody wants to help anyway :-) 

I want to create some tests for a site with a self-signed certificate, so I need to pass the "ignoreHTTPSErrors=True" option.
At the moment, I use this (as a workaround):

```
def test_invalid_user_login(browser):
    context=browser.newContext(ignoreHTTPSErrors=True)
    page=context.newPage()
    login(page,username="invaliduser",password="validpassword")
```
This works, but would require too redundant line per test. Not a big problem, but I think not the best solution. 

From the README.MD, I got this information:

> browser_context_args: Override the options for browser.newContext(). It should return a Dict.

Which sound exactly what I need. But all ways I tried, I got still an certificate error --> ignoreHTTPSErrors=True is not used.
`FAILED test_1.py::test_normal_login[chromium] - playwright.helper.Error: net::ERR_CERT_AUTHORITY_INVALID at http://10.10.10.10/
`
Can give somebody me a correct example, how to set/use this fixture?

Thanks a lot

--- a/README.md
+++ b/README.md
@@ -120,6 +120,40 @@ def test_visit_example(page):
     # -> Will result in http://localhost:8080/admin
 ```
 
+#### Ignore HTTPS errors
+
+conftest.py
+
+```py
+import pytest
+
+@pytest.fixture(scope="session")
+def browser_context_args(browser_context_args):
+    return {
+        **browser_context_args,
+        "ignoreHTTPSErrors": True
+    }
+```
+
+
+#### Use custom viewport size
+
+conftest.py
+
+```py
+import pytest
+
+@pytest.fixture(scope="session")
+def browser_context_args(browser_context_args):
+    return {
+        **browser_context_args,
+        "viewport": {
+            "width": 1920,
+            "height": 1080,
+        }
+    }
+```
+
 ## Debugging
 
 #### Use with pdb

--- a/pytest_playwright/pytest_playwright.py
+++ b/pytest_playwright/pytest_playwright.py
@@ -130,14 +130,14 @@ def _handle_page_goto(
     url = args.pop()
     if not (url.startswith("http://") or url.startswith("https://")):
         url = base_url + url
-    return page._goto(url, *args, **kwargs)
+    return page._goto(url, *args, **kwargs)  # type: ignore
 
 
 @pytest.fixture
 def page(context: BrowserContext, base_url: str) -> Generator[Page, None, None]:
     page = context.newPage()
-    page._goto = page.goto
-    page.goto = lambda *args, **kwargs: _handle_page_goto(
+    page._goto = page.goto  # type: ignore
+    page.goto = lambda *args, **kwargs: _handle_page_goto(  # type: ignore
         page, list(args), kwargs, base_url
     )
     yield page

